modules/client/src/addressList/internal/client/methods.ts

```diff
@@ -79,14 +78,14 @@ export class ClientAddressListMethods extends ClientCore
       signer,
     );
 
-    // TODO: Compute the cid instead of pinning the data on the cluster
-    let cid = "";
+    const startTimestamp = params.startDate?.getTime() || 0;
+    const endTimestamp = params.endDate?.getTime() || 0;
 
     const tx = await addresslistContract.createVote(
-      toUtf8Bytes(cid),
```

Comment:
I would keep the pinning feature here... until we do the split. Removing it serves no purpose IMO, as we're about to have `getDao()` reading this in short